testing/test_pytest_html.py

```diff
@@ -1005,12 +1005,31 @@ def test_css(self, testdir, recwarn, colors):
             assert str(v["path"]) in html
             assert v["style"] in html
 
-    def test_css_invalid(self, testdir, recwarn):
+    @pytest.mark.parametrize(
+        "file1,file2,expected",
+        [
+            ("style.css", "", "Missing CSS files: style.css,"),
+            ("abc.css", "xyz.css", "Missing CSS files: abc.css, xyz.css"),
+        ],
+    )
+    def test_css_invalid(self, testdir, recwarn, file1, file2, expected):
```

Comment:
This is correct, but you could make it even more clean by parametrizing with a list of files instead of passing `file1` and `file2`. For example:
```
$ cat test_parametrize.py
import pytest

@pytest.mark.parametrize("arg", ["arg1", "arg2", ["arg3.a", "arg3.b"]])
def test_list(arg):
     assert arg is not None
$
```

In the example above the test is run 3 times. First with `"arg1"`, then `"arg2"`, and finally with the list `["arg3.a", "arg3.b"]`

This way your test can be expanded on in the future without changing its signature every time, and it becomes easier to reason about